app/config/parameters.yml.dist

```diff
@@ -8,12 +8,21 @@ parameters:
     database_path:     "%kernel.root_dir%/app.sqlite"
 
     phpcr_backend:
-        # use Jackalope Doctrine DBAL
+        # if you're using Jackalope Doctrine DBAL
         type: doctrinedbal
         connection: default
         caches:
             meta: doctrine_cache.providers.phpcr_meta
             nodes: doctrine_cache.providers.phpcr_nodes
+        
+        # if you're using Jackalope Jackrabbit, comment the lines before
+        # and uncomment the lines after this comment
+        #logging: true
+        #profiling: true
+        #type: jackrabbit
+        #url: http://localhost:8080/server/
+        #parameters:
+        #    jackalope.check_login_on_server: false
```

Comment:
I think we can soon remove it for both. However the lazy loading is not yet released for DBAL:
https://github.com/jackalope/jackalope-doctrine-dbal/pull/242

We hopefully soon have 1.2 released and then we should check if it works even with this setting set to false when initializing a new project with DBAL. right now if this is set to true, it will fail afaik since before initializing this will explode.